### PR TITLE
Auto-format in devcontainer

### PR DESCRIPTION
**Pull request by SashaWeinstein**
Small PR, just requires one reviewer 🍰 
Figured out this fix on the QAQC app first, also applied it here

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -25,3 +25,4 @@ ENV PATH="~/.poetry/bin:$PATH"
 ENV PATH="~/.local/bin:$PATH"
 
 RUN /usr/local/bin/python3 -m pip install -U flake8
+RUN /usr/local/bin/python3 -m pip install -U black
```
Comment (mbh329) [inline .devcontainer/Dockerfile:28]: Taking a look at the qaqc dockerfile, we install bandit and not flake8. I think it makes sense to stay as consistent as possible

**Review by mbh329 — CHANGES_REQUESTED** [review]

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (mbh329) [reply .devcontainer/Dockerfile:28]: Rebuilding the dev container and it's taking ages, is this true for you as well @SashaWeinstein

Comment (SashaWeinstein) [reply .devcontainer/Dockerfile:28]: started rebuilding now

Comment (mbh329) [reply .devcontainer/Dockerfile:28]: got it finally

Comment (SashaWeinstein) [reply .devcontainer/Dockerfile:28]: Mine rebuilt pretty quick, meant to look at clock but within 5 minutes I think. Max when was the last time you turned off your computer? 
Also made the requested change from `flake8` to `bandit`

Comment (mbh329) [reply .devcontainer/Dockerfile:28]: yesterday. great will rebuild.

**Review by mbh329 — APPROVED** [review]

**Pull request merged**